Title: Louis F. Gil Acevedo 
Date: 12-13-2017
Modified: 1-18-2017
Slug: louis
Category: People
Tags: de-novo de-bruijn

## Contact info:

 - e-mail - <louis-gil@live.com>
 - Github - <https://github.com/LouisGil/>

# Bio:
Hi Im Louis Im a Computer Science undergraduate student. Im interested in the beautiful merger of computer science and biology. It all started with [the fibonacci numbers](https://en.wikipedia.org/wiki/Fibonacci_number).

> Insert Quotes here


# Weekly Updates


### During this semester we'll be working on a shared transcript discovery tool that creates a de-bruijn graph from different transcripts in order to analyze differential expression.

###
### Week 8 ():
- Fixing Errors when running fluxsimulatar.
- This folder contains both Fastq files simulated from yeast (Readme contains step by step guide on the run)https://drive.google.com/open?id=1qm2Cay0_CcidLqT6CaN_7btRWn8Bb5OZ

### Week 7 ():
-Simulate 2 RNA sequences using flux-simulator saccharomyces cerevisiae(yeast), to test the DBGDE.

### Week 6 (7-12):
- Add Diferential Expresion to current vertion. Such that by a user set cuttoff point, links with a lower score than this will be deleted. 

### Week 3: (11-14/Dec)
- Add kmer count to output file.
-[gfalint](https://github.com/sjackman/gfalint)
-[gfapy](https://github.com/ggonnella/gfapy)
-[exampple gfa 2](https://github.com/sjackman/gfalint/blob/master/examples/big1.gfa)

### Week 2: (4-8/Dec)
- Modify [dbg](https://github.com/pmelsted/dbg) to accept two files and check for contigs correctness.

### Week 1: (27/Nov-1/Dec)
- Attack plan, and establish goals
- Test [dbg](https://github.com/pmelsted/dbg) for correctness by running the same fasta file and check for difference, and check the dictionary used for correct kmer count.
- Read sugested papers
  - [Heuristic pairwise alignment of de Bruijn graphs](https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-16-S11-S5)
  - [Transcriptomic changes during regeneration of the central nervous system in an echinoderm](https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-357)
  - [Salmon provides accurate, fast, and bias-aware transcript expression estimates using dual-phase inference](https://www.biorxiv.org/content/early/2016/08/30/021592)



